sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -2407,6 +2450,14 @@ public void merge(final @NotNull ExternalOptions options) {
     if (options.isSendModules() != null) {
       setSendModules(options.isSendModules());
     }
+
+    if (options.isEnableAutomaticCheckIns() != null) {
+      setEnableAutomaticCheckIns(options.isEnableAutomaticCheckIns());
+    }
+    if (options.getIgnoredCheckIns() != null) {
+      final List<String> ignoredCheckIns = new ArrayList<>(options.getIgnoredCheckIns());
+      setTracePropagationTargets(ignoredCheckIns);
```

Comment:
Probably CopyPaste error. Ignored check-ins shouldn't be added to `tracePropagationTargets`, instead `setIgnoredCheckIns` should be called

Reply:
good catch! Seems I missed adding tests to `SentryOptionsTest` it seems. adding those as they should also catch this